network/syncer/common_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 The Swarm Authors
```

Comment:
Having files and packages named `common` is an anti-pattern. I suggest we move `newBzzSyncWithLocalstoreDataInsertion` where it is used.